api/src/main/java/com/homeaway/datapullclient/process/DataPullTask.java

```diff
@@ -123,7 +124,25 @@ private void runSparkCluster() {
             for (ClusterSummary cluster : listClustersResult.getClusters()) {
                 if (cluster.getName().matches(".*-emr-.*-pipeline")) {
                     ListStepsRequest listSteps = new ListStepsRequest().withClusterId(cluster.getId());
-                    ListStepsResult steps = emr.listSteps(listSteps);
```

Comment:
the following change is for AWS ElasticMapReduce reach rate limit. it will retry 10 times